packages/ui/cypress/tests/transactions.cy.ts

```diff
@@ -0,0 +1,66 @@
+import { injectedAccounts } from '../fixtures/injectedAccounts'
+import { landingPageUrl } from '../fixtures/landingData'
+import { landingPage } from '../support/page-objects/landingPage'
+import { multisigPage } from '../support/page-objects/multisigPage'
+import { notifications } from '../support/page-objects/notifications'
+import { sendTxModal } from '../support/page-objects/sendTxModal'
+import { topMenuItems } from '../support/page-objects/topMenuItems'
+
+const AliceAddress = Object.values(injectedAccounts)[0].address
+
+describe('Perform transactions', () => {
+  beforeEach(() => {
+    cy.visit(landingPageUrl)
+    cy.initExtension(injectedAccounts)
+    topMenuItems.connectButton().click()
+    landingPage.accountsOrRpcLoader().should('contain', 'Loading accounts')
+    cy.getAuthRequests().then((authRequests) => {
+      const requests = Object.values(authRequests)
+      // we should have 1 connection request to the extension
+      cy.wrap(requests.length).should('eq', 1)
+      // this request should be from the application Multix
+      cy.wrap(requests[0].origin).should('eq', 'Multix')
+      // let's allow it for Alice
+      cy.enableAuth(requests[0].id, [AliceAddress])
+      // the ui should then move on to connecting to the rpcs
+      topMenuItems.multiproxySelector().should('be.visible')
+    })
+  })
+
+  it('Abort a tx with Alice', () => {
+    multisigPage.newTransactionButton().click()
+    sendTxModal.sendTxTitle().should('be.visible')
+    sendTxModal.fieldTo().click().type(`${AliceAddress}{enter}`)
+    sendTxModal.fieldAmount().click().type('0.001')
+    sendTxModal.buttonSend().should('be.enabled').click()
+    cy.getTxRequests().then((req) => {
```

Comment:
I see that you created a class Extension and all the interactions will be mocked as we don't want to deal with the real Polkadot plugin. Is it a correct understanding?

Reply:
Yes, that's the idea behind the "Cypress plugin" we're building, because you can't install plugins in Cypress. This logic will end up in a separate repo, published to NPM, and imported here, so that others can use it.